migration/idempotent/011-maintenance.sql

```diff
@@ -458,6 +490,10 @@ AS $$
 DECLARE
    startT TIMESTAMPTZ;
 BEGIN
+    -- Note: We cannot use SET in the procedure declaration because we do transaction control
```

Comment:
I don't understand this one. Where are we doing transaction control in the body of this procedure? Does it apply transitively to other procedures called from this one?

Reply:
yes it applies transitively